### PR TITLE
fixed typos throughout endpoints and readme

### PR DESCRIPTION
**Pull request by CamMeiners**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue): Typos in a few places in endpoints, updated readme file, investigated endpoints to find ones that aren't working/haven't been built. 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by nforman91 — APPROVED** [review]
Thanks for fixing the typos! Remember to add links to the Trello card & Loom video

File: api/app.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ app.use(['/parent', '/parents'], parentRouter);
 app.use(['/instructor', '/instructors'], instructorRouter);
 app.use(['/user'], userRouter);
 // app.use(['/inbox', '/inboxes'], inboxRouter);
-app.use(['/course-type', '/course-types'], courseTypesRouter);
+app.use(['/course-type', '/course-types', '/course'], courseTypesRouter);
```
Comment (RoseGabriely) [inline api/app.js]: Could you also add /courses?

File: README.md [hunk 2]
```diff
@@ -82,193 +82,204 @@ Updated table Source: https://dbdesigner.page.link/WTZRbVeTR7EzLvs86
...
+#### Class:
+
+| Method | URL |
+Description |
+| ----- | ------------------- | --------------------------------------------------- | ----------- |
+| [GET] | /class-instance | Returns an array filled with specific class objects |
+| [GET] | /class-instance/:class-id | Returns the class with specified `id`.  
+| [POST] | /class-instance | Returns the class object with the specified `id`. Creates a class.
+| [PUT] | /class-instance/:class-id | Returns the class object with the specified `id`. Updates a specific class.
+| [DELETE]| /class-instance/:class-id | Returns the class object with the specified `id`. Deletes specific class.
```
Comment (RoseGabriely) [inline README.md]: Thanks for adding this! Could we style this a bit better so that the table columns align?

**Review by RoseGabriely — CHANGES_REQUESTED** [review]
Lookin good so far! Keep going :)

File: README.md (hunk 2 above)
Comment (CamMeiners) [reply README.md]: Done

File: api/app.js (hunk 1 above)
Comment (CamMeiners) [reply api/app.js]: done

**Review by RoseGabriely — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -82,193 +82,202 @@ Updated table Source: https://dbdesigner.page.link/WTZRbVeTR7EzLvs86
...
-| [POST]   | /newsfeed/    | Returns the event object with the specified `id`. Creates a new obect.                                     |
-| [PUT]    | /newsfeed/    | Returns the event object with the specified `id`. Updates specific object.                                 |
-| [DELETE] | /newsfeed/:id/| Returns the event object with the specified `id`. Deletes specific object.                                 |
+| Method   | URL            | Description                                                                |
+| -------- | -------------- | -------------------------------------------------------------------------- |
+| [GET]    | /newsfeed/     | Returns an array filled with event objects.                                |
+| [GET]    | /newsfeed/:id/ | Returns the event object with the specified `id`.                          |
+| [POST]   | /newsfeed/     | Returns the event object with the specified `id`. Creates a new obect.     |
+| [PUT]    | /newsfeed/     | Returns the event object with the specified `id`. Updates specific object. |
+| [DELETE] | /newsfeed/:id/ | Returns the event object with the specified `id`. Deletes specific object. |
```
Comment (katherineyevsukov) [inline README.md:282]: Readme formatting updates looks good

File: README.md [hunk 4]
```diff
@@ -82,193 +82,202 @@ Updated table Source: https://dbdesigner.page.link/WTZRbVeTR7EzLvs86
...
 
 <br />
```
Comment (mcmillanj) [inline README.md:283]: Read had minor changes that were consistent with database and formatting

File: __tests__/routes/newsfeed.test.js [hunk 5]
```diff
@@ -73,9 +73,9 @@ describe('newsfeeds router endpoints', () => {
...
       Newsfeed.updateNewsfeed.mockResolvedValue([newsfeed]);
 
-      const res = await request(server).put('/newsfeed').send(newsfeed);
+      const res = await request(server).put('/newsfeed/1').send(newsfeed);
       expect(res.status).toBe(200);
-      expect(res.body.newsfeed.description).toBe('Testing PUT newsfeed');
+      expect(res.body).toBe(1);
       expect(Newsfeed.updateNewsfeed.mock.calls.length).toBe(1);
     });
   });
```
Comment (katherineyevsukov) [inline __tests__/routes/newsfeed.test.js:81]: Updates make sense. Status codes are now correct

**Review by katherineyevsukov — APPROVED** [review]
Great job catching all the necessary syntax and formatting changes. Routes are much more logical and clear now. Looks like a lot of work was done and it all looks great.

**Review by mcmillanj — APPROVED** [review]
approved, mostly syntacs changes and a few route fillens. but all look great

**Pull request merged**